### ISSUE TITLE
IRC: the fields in theewserver dialogueareunlabelled for screenreaders

### ISSUE THREAD

**Issue by georgehazan** [miranda-ng/miranda-ng#353]
**Reported by Piciok on 28 Dec 2014 23:15 UTC**
When you go to Options--Protocolls -- your IRC account-- Account-- Add - the fields there are not labelled and screenreaders can't telll what the fields are for

**Comment by JeanPaulLucien** [comment]
I did not find these options.
